utest.h

```diff
@@ -736,6 +741,12 @@ UTEST_WEAK int utest_main(int argc, const char *const argv[]) {
     } else if (0 ==
                utest_strncmp(argv[index], output_str, strlen(output_str))) {
       utest_state.output = utest_fopen(argv[index] + strlen(output_str), "w+");
+    } else if (0 == utest_strncmp(argv[index], list_str, strlen(list_str))) {
+      for (index = 0; index < utest_state.tests_length; ++index) {
+        puts(utest_state.tests[index].name);
+      }
+      /* When printing the test list, don't actually run the tests */
+      exit(EXIT_SUCCESS);
```

Comment:
I'd rather this was just a return 0, just because there might be uses of utest_main that we don't control and we don't want to decide their exit for them.

Reply:
good point. Done